src/Router/Router.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Router;
+
+use App\Controllers\article;
+use App\Controllers\Homepage;
+use App\controllers\Articles;
```

Comment:
Don't forget the uppercase C on Controller (and uppercase A on Article)

Reply:
Done